src/js/components/balances.js

```diff
@@ -1377,15 +1377,15 @@ function SweepModalViewModel() {
             PENDING_ACTION_FEED.add(sendTxHash, "sends", sendData);
 
             // here we adjust the BTC balance whith the change output
-            if (selectedAsset.ASSET != 'BTC') {
+            if (selectedAsset.ASSET !== KEY_ASSET.BTC) {
               var newBtcBalance = CWBitcore.extractChangeTxoutValue(sendData.source, unsignedTxHex);
-              $.jqlog.debug("New BTC balance: " + newBtcBalance);
+              $.jqlog.debug("New ' + KEY_ASSET.BTC + ' balance: " + newBtcBalance);
```

Comment:
need double quotes here

Reply:
Thank you for your review.  I'll fix them and re-push within a few days.